internal/server/firm_deputy_hub.go

```diff
@@ -3,29 +3,45 @@ package server
 import (
 	"github.com/ministryofjustice/opg-sirius-supervision-firm-deputy-hub/internal/sirius"
 	"net/http"
+	"strconv"
+	"strings"
 )
 
 type FirmDeputyHubInformation interface {
+	GetFirmDetails(sirius.Context, int) (sirius.FirmDetails, error)
 }
 
 type firmDeputyHubVars struct {
 	Path      string
 	XSRFToken string
 	Error     string
 	Errors    sirius.ValidationErrors
+	FirmDetails sirius.FirmDetails
 }
 
-func renderTemplateForFirmDeputyHub(client FirmDeputyHubInformation, tmpl Template) Handler {
+func renderTemplateForFirmHub(client FirmDeputyHubInformation, tmpl Template) Handler {
 	return func(perm sirius.PermissionSet, w http.ResponseWriter, r *http.Request) error {
 		if r.Method != http.MethodGet {
 			return StatusError(http.StatusMethodNotAllowed)
 		}
 
 		ctx := getContext(r)
+		url := r.URL.Path
+		idFromParams := strings.Trim(url, "/")
+		if idFromParams == "" {
+			idFromParams = "0"
```

Comment:
In what situation would an id not be present and what does it mean for the firm id to be 0?

Reply:
Ah I think I forgot to remove this